linux_os/guide/services/fapolicyd/configure_fapolicyd_mounts/rule.yml

```diff
@@ -27,6 +27,7 @@ identifiers:
 
 references:
     ospp: FPT_SRP_EXT.1
+    srg: SRG-OS-000480-GPOS-00232,SRG-OS-000370-GPOS-00155
```

Comment:
Nack on ``SRG-OS-000480-GPOS-00232``, since that control is about (web) application firewalls. 

Need to explore ``SRG-OS-000370-GPOS-00155`` further. Is the default for fapolicyd truly a "deny-all, permit-by-exception" policy?

Reply:
removed SRG-OS-000480-GPOS-00232

fapolicy is application whitelisting (http://people.redhat.com/sgrubb/fapolicyd/) so I think that would apply to SRG-OS-000370-GPOS-00